### ISSUE TITLE
Documentation: 

### ISSUE THREAD

**Issue by techie2000** [NixOS/nixpkgs#334845]
## Problem

The link to https://asus-linux.org/asusctl/ in https://github.com/NixOS/nixpkgs/blob/nixos-unstable/nixos/modules/services/hardware/asusd.nix is broken (returns a 404 error).

## Proposal

Browsing around (https://asus-linux.org/) the best replacement (guessing what was originally displayed) is https://gitlab.com/asus-linux/asusctl.

## Checklist

- [x] checked [latest Nixpkgs manual] \([source][nixpkgs-source]) and [latest NixOS manual] \([source][nixos-source])
- [x] checked [open documentation issues] for possible duplicates
- [x] checked [open documentation pull requests] for possible solutions

[latest Nixpkgs manual]: https://nixos.org/manual/nixpkgs/unstable/
[latest NixOS manual]: https://nixos.org/manual/nixos/unstable/
[nixpkgs-source]: https://github.com/NixOS/nixpkgs/tree/master/doc
[nixos-source]: https://github.com/NixOS/nixpkgs/tree/master/nixos/doc/manual
[open documentation issues]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+label%3A%229.needs%3A+documentation%22
[open documentation pull requests]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+documentation%22%2C%226.topic%3A+documentation%22

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by techie2000** [comment]
cc @K900 @aacebedo

**Comment by Masrkai** [comment]
that's correct going to `https://asus-linux.org/` provides a link to the Gitlab page of the project
